docker-compose-with-zipkin.yaml

```
@@ -0,0 +1,29 @@
+version: '3.1'
+
+services:
+  db:
+    image: mysql:8.0.37-debian
+    restart: always
+    environment:
+        MYSQL_ROOT_PASSWORD: root
+    ports:
+      - "3306:3306"
+
+  zipkin:
+    image: openzipkin/zipkin
+    ports:
+      - 9411:9411
+
+  jenkins:
+    image: theweatherman/jenkins:otel
+    ports:
```

Can be use the official Jenkins image please especially since we're already copying in the agent?

Shouldn't this have a build statement:

```suggestion
    build:
      context: .
    ports:
```